Dockerfile

```diff
@@ -1,3 +1,18 @@
+# Git info stage
+FROM alpine/git:2.49.0 as git-info
+WORKDIR /app
+COPY . .
+COPY .git ./.git
```

Comment:
Isn't the `COPY .git ./.git` superfluous? 
`.git` has been removed from the `.dockerignore`, so the `COPY . .` should already copy everything in